.github/ISSUE_TEMPLATE/crash-report.yml

```diff
@@ -0,0 +1,107 @@
+    validations:
+      required: true
+  - type: dropdown
+    id: jasp-module
+    attributes:
+      label: JASP Module
+      description:  Is there a specific JASP module that caused the crash?
+      multiple: true
+      options:
+        - Unrelated
+        - Acceptance Sampling
+        - ANOVA
+        - Audit
+        - Bain
+        - Bayes Factor Functions
+        - BFpack
+        - Bsts
+        - Circular Statistics
+        - Cochrane Meta-Analyses
+        - Descriptives
+        - Distributions
+        - Equivalence T-Tests
+        - Factor
+        - Frequencies
+        - JAGS
+        - Learn Bayes
+        - Learn Stats
+        - Machine Learning
+        - Meta Analysis
+        - Mixed Models
+        - Network
+        - Power
+        - Predictive Analytics
+        - Process
+        - Prophet
+        - Quality Control
+        - Regression
+        - Reliability
+        - Robust T-Tests
+        - SEM
+        - Survival
+        - Summary Statistics
+        - T-Tests
+        - Time Series
+        - Visual Modeling
+        - Other
+    validations:
+      required: true
+  - type: input
+    id: jasp-analysis
+    attributes:
+      label: Is there a specific analysis that caused the crash?
+      description: Please mention the name of the analysis or method that the causes the issue.
+      placeholder: "One Sample T-Test"
+    validations:
+      required: false
+  - type: textarea
+    id: crash-description
+    attributes:
+      label: Crash Description
+      description: |
+        Please tell us what happened.
+        
+        Note: You can attach images or log files by clicking this area to highlight it and then dragging files in. If you upload a JASP file, you need to compress your .jasp-file before uploading it to GitHub (aka as a .zip-file; for additional help, see https://jasp-stats.org/2018/03/29/request-feature-report-bug-jasp/). In addition, if you would prefer to not make your data publicly available, you can send your files directly to us, issues@jasp-stats.org.
+      placeholder: "I clicked on bootstrap and JASP crashed."
+    validations:
+      required: true
+  - type: textarea
+    id: log
+    attributes:
+      label: Log (if any)
+      description:
+        Please upload your Log files by dropping them into the area below, and add any information notes if necessary.
+        
+        **Read our [Logging Guide](https://github.com/jasp-stats/jasp-desktop/blob/stable/Docs/user-guide/logging-howto.md) page to learn where to (and how to) find JASP log files.**
+    validations: 
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: Please tell us what happened. Can you reproduce the crash? If so, please let us know what steps you take to reproduce this bug.
+      value: |
```

Comment:
I guess it's would be better if it's a placeholder? because people can just submit with default text😜

Reply:
true, fixed it for bug and crash report